### ISSUE TITLE
Can't use AlphabeticIndex

### ISSUE THREAD

**Issue by Hultner** [ovalhub/pyicu#89]
I'm trying to use [icu::AlphabeticIndex](http://icu-project.org/apiref/icu4c/classicu_1_1AlphabeticIndex.html#details) in order to create a character index map to be used in a UI but I'm unable to import it using PyICU.

```python
bpython version 0.17.1 on top of Python 3.7.0 /usr/local/opt/python/bin/python3.7
>>> import icu
>>> icu.ICU_VERSION
'62.1'
>>> from icu import AlphabeticIndex
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    from icu import AlphabeticIndex
ImportError: cannot import name 'AlphabeticIndex' from 'icu' (/Users/hultner/.local/share/virtualenv
s/…/lib/python3.7/site-packages/icu/__init__.py)
```

How am I supposed to call this class? 
Is there another interface to reach it through? 

I've tried using `getExemplarSet` and sort it using the collator in order to create the index that way instead but I end up with characters which aren't used as index-characters in targeted languages (e.g. é and à for Swedish which should be regarded as e and a, while å, ä and ö is standalone characters).

**Comment by ovalhub** [comment]
On Thu, 13 Sep 2018, Alexander Hultnér wrote:

> I'm trying to use [icu::AlphabeticIndex](http://icu-project.org/apiref/icu4c/classicu_1_1AlphabeticIndex.html#details) in order to create a character index map to be used in a UI but I'm unable to import it using PyICU.
>
> ```python
> bpython version 0.17.1 on top of Python 3.7.0 /usr/local/opt/python/bin/python3.7
>>>> import icu
>>>> icu.ICU_VERSION
> '62.1'
>>>> from icu import AlphabeticIndex
> Traceback (most recent call last):
>  File "<input>", line 1, in <module>
>    from icu import AlphabeticIndex
> ImportError: cannot import name 'AlphabeticIndex' from 'icu' (/Users/hultner/.local/share/virtualenv
> s/?/lib/python3.7/site-packages/icu/__init__.py)
> ```
>
> How am I supposed to call this class?
> Is there another interface to reach it through?
>
> I've tried using `getExemplarSet` and sort it using the collator in order to create the index that way instead but I end up with characters which aren't used as index-characters in targeted languages (e.g. é and à for Swedish which should be regarded as e and a, while å, ä and ö is standalone characters).

I'd never heard of this class before thus the wrapper for it was missing.
I now added a wrapper for AlphabeticIndex and ImmutableIndex (Bucket is 
just a tuple) in HEAD at github: https://github.com/ovalhub/pyicu

Please, see the bottom of the test_Collator.py for a trivial example of 
usage. AlphabeticIndex and ImmutableIndex are iterators over their
buckets. The get/set functions that took only zero/one parameter were 
implemented as python get/set properties.
For an overview of what's available, please see here:
   https://github.com/ovalhub/pyicu/blob/master/collator.cpp#L157

Please, let me know if this works for you.
Thanks !

Andi..

**Comment by Hultner** [comment]
Hi Andi, 
Thanks for swift response, when can we expect this to be included in the pip package?

**Comment by ovalhub** [comment]
> On Sep 13, 2018, at 23:46, Alexander Hultnér <notifications@github.com> wrote:
> 
> Hi Andi,
> Thanks for swift response, when can we expect this to be included in the pip package?
> 
Have you tried it out yet ?

Andi..

**Comment by Hultner** [comment]
Hi Andi/@ovalhub 

Sorry for the delayed answer, I missed that you had answered. I tried it out today again (were refactoring the code where we use ICU) and I still can't use the AlphabeticIndex. 

I've currently implemented my own AlphabeticIndex in python using the Exemplar sets but would love to remove it in favour of the native more battle-tested class. 


Here's my result using py-icu 2.1 on python 3.7. 

```python
 » bpython
bpython version 0.17.1 on top of Python 3.7.0 /usr/local/opt/python/bin/python3.7
>>> from icu import AlphabeticIndex
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    from icu import AlphabeticIndex
ImportError: cannot import name 'AlphabeticIndex' from 'icu' (/Users/hultner/.local/share/virtualenv
s/guestlist-ktnm5Z_6/lib/python3.7/site-packages/icu/__init__.py)
>>> import icu
>>> icu.VERSION
'2.1'
>>>
>>> from icu import *
>>> index = AlphabeticIndex(Locale.getItaly())
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    index = AlphabeticIndex(Locale.getItaly())
NameError: name 'AlphabeticIndex' is not defined
>>>
```

**Comment by ovalhub** [comment]
> On Oct 4, 2018, at 01:28, Alexander Hultnér <notifications@github.com> wrote:
> 
> Hi Andi/@ovalhub
> 
> Sorry for the delayed answer, I missed that you had answered. I tried it out today again (were refactoring the code where we use ICU) and I still can't use the AlphabeticIndex.
> 
> I've currently implemented my own AlphabeticIndex in python using the Exemplar sets but would love to remove it in favour of the native more battle-tested class.
> 
> Here's my result using py-icu 2.1 on python 3.7.
> 
AlphabeticIndex is available from the HEAD revision at github, not from any released version of PyICU yet.
Go to github's pyicu page and build pyicu.
From the command line in a shell window:
$ git clone <the pyicu repository>
$ cd pyicu (into directory with setup.py)
$ python setup.py build
$ sudo python setup. py install

I can't give you the exact commands as I'm not near my keyboard at the moment.

Andi..

**Comment by ovalhub** [comment]
This is available in PyICU 2.2
